src/tactic/lean_core_docs.lean

```diff
@@ -0,0 +1,650 @@
+
+/--
+This tactic applies to a goal whose target has the form `t ~ u` where `~` is a reflexive relation,
+that is, a relation which has a reflexivity lemma tagged with the attribute `[refl]`.
+The tactic checks whether `t` and `u` are definitionally equal and then solves the goal.
+-/
+add_tactic_doc
+{ name       := "refl",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.refl, `tactic.interactive.reflexivity],
+  tags       := ["core", "basic", "finishing"] }
+
+add_tactic_doc
+{ name       := "rename",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.rename],
+  tags       := ["core", "renaming"] }
+
+add_tactic_doc
+{ name       := "repeat",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.repeat],
+  tags       := ["core", "combinator"] }
+
+add_tactic_doc
+{ name       := "revert",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.revert],
+  tags       := ["core", "context management", "goal management"] }
+
+add_tactic_doc
+{ name       := "right",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.right],
+  tags       := ["core", "basic", "logic"] }
+
+/--
+`rw e` applies an equation or iff `e` as a rewrite rule to the main goal. If `e` is preceded by
+left arrow (`←` or `<-`), the rewrite is applied in the reverse direction. If `e` is a defined
+constant, then the equational lemmas associated with `e` are used. This provides a convenient
+way to unfold `e`.
+
+`rw [e₁, ..., eₙ]` applies the given rules sequentially.
+
+`rw e at l` rewrites `e` at location(s) `l`, where `l` is either `*` or a list of hypotheses
+in the local context. In the latter case, a turnstile `⊢` or `|-` can also be used, to signify
+the target of the goal.
+-/
+add_tactic_doc
+{ name       := "rewrite / rw",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.rw, `tactic.interactive.rewrite],
+  tags       := ["core", "basic", "rewriting"] }
+
+add_tactic_doc
+{ name       := "rwa",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.rwa],
+  tags       := ["core", "rewriting"] }
+
+add_tactic_doc
+{ name       := "show",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.show],
+  tags       := ["core", "goal management", "renaming"] }
+
+add_tactic_doc
+{ name       := "simp_intros",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.simp_intros],
+  tags       := ["core", "simplification"] }
+
+add_tactic_doc
+{ name       := "skip",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.skip],
+  tags       := ["core", "combinator"] }
+
+add_tactic_doc
+{ name       := "solve1",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.solve1],
+  tags       := ["core", "combinator", "goal management"] }
+
+add_tactic_doc
+{ name       := "sorry",
```

Comment:
```suggestion
{ name       := "sorry / admit",
```

Reply:
This looks good -- the doc string for `sorry` applies to both.